app/store/slices/city.js

```diff
@@ -0,0 +1,59 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+
+
+export const fetchData = createAsyncThunk(
+  'city/get',
+  async(cityName) => {
+    let url = 'https://api.openweathermap.org/data/2.5/forecast?q=' + cityName + '&appid=75f4785546ba2c0a068da88d46e4db01'
```

Comment:
would be a good idea to have appid as a variable because this is a value that could change